### PR TITLE
Use bulk fixer endpoints and improve feedback with notices

### PR DESCRIPTION
**Pull request by miguelxpn**
#### Changes proposed in this Pull Request

* Use bulk fixer endpoints instead of the individual ones.
* Improve UI feedback for fixers by pooling the bulk fixer status endpoint and showing a notice in case of success or error.

#### Testing instructions

* Set up a JP site with Scan and add a few threats (You can use [this plugin](https://github.com/Automattic/jetpack-threat-tester) to add a few innocuous threats for testing).
* Go to the scan section of both Calypso Blue and Calypso Green (To test Calypso Green you need to run `yarn start start-jetpack-cloud` and then go to `http://jetpack.cloud.localhost:3000` on your browser)  and try out the fix all button and verify that a single network call was made.
* Verify that a success/error notice appears when all fixers finish their processing.

#### Screenshots 
Before we showed no feedback in case the fixer didn't work:
error-old
After we start showing a notice in case one of the fixers failed:
error

### CODE REVIEW

**Comment by seear** [comment]
> Set up a JP site with Scan and add a few threats.

Do we have an easy way to set up a site with a few fixable threats?

> Try out the fix all button and verify that a single network call was made.

Some links would be helpful here. Am I testing this that cloud.jetpack.com, or wordpress.com, or both?

**Comment by seear** [comment]
I actually couldn't find a satisfactory example of good detailed testing instructions in the first couple of pages of open Calypso PRs, so adding my own here. This is roughly the level of detail I would like to see if I was coming to this PR without deep knowledge of the feature area:

* create a Jurassic Ninja site at https://jurassic.ninja/create/?shortlived
* Upload and activate the [jetpack-threat-tester](https://github.com/Automattic/jetpack-threat-tester) plugin on the Jurassic Ninja site
* Activate some threats from the wp-admin/tools menu
* Connect the site to Jetpack and purchase a plan that includes scan with credits

## Testing wordpress.com

* Go to https://hash-bc8a5adf2021a0492d342bd2066283c25657e603.calypso.live/scan/{test site}
EXPECTED: you should see some threats listed
* Open the network debug area in your browser
* Click the fix all button
EXPECTED: Only a single network call should be made
EXPECTED: A success OR error notification should be shown. An error is fine—some threats cannot be fixed.

## Testing cloud.jetpack.com 

* Checkout and build this branch locally
* Go to {local testing path, not sure what this is}/scan/{test site}
* Verify as for wordpress.com above

File: client/state/data-layer/wpcom/sites/alerts/fix-status.js
Comment (seear) [inline client/state/data-layer/wpcom/sites/alerts/fix-status.js:18]: This might be `POLL_EVERY_MILLISECONDS` or possibly `POLL_EVERY_SECOND`?

File: client/state/data-layer/wpcom/sites/alerts/fix-status.js
Comment (seear) [inline client/state/data-layer/wpcom/sites/alerts/fix-status.js:65]: Does it make sense to talk about "selected threats"? As I understand it, the user can fix either one threat, or all threats?

Comment (seear) [reply client/state/data-layer/wpcom/sites/alerts/fix-status.js:65]: > Please contact our support

"Please contact support" is in more common use, I think. Should it link to the Jetpack support page?

**Review by seear — APPROVED** [review]
Nice work, code looks absolutely fine, just a couple of minor things with the copy. Looks good to merge when you are happy with those.

File: client/state/data-layer/wpcom/sites/alerts/fix-status.js
Comment (miguelxpn) [reply client/state/data-layer/wpcom/sites/alerts/fix-status.js:18]: Whoops, silly typo. Thanks!

File: client/state/data-layer/wpcom/sites/alerts/fix-status.js
Comment (miguelxpn) [reply client/state/data-layer/wpcom/sites/alerts/fix-status.js:65]: I was trying to come up with a generic phrase that could fit in both cases. But perhaps it would be better to check if it's a single threat being fixed and adjust the message accordingly.

**Review by rcanepa — APPROVED** [review]
This is an awesome improvement. Everything works nicely, and the code looks great.

Thanks for working on this!

**Review by rcanepa — APPROVED** [review]
I just gave this PR a second review and it's definitely still working great.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5109465

Thank you @miguelxpn for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5109465).

**Pull request merged**
